### DISCUSSION TITLE
What am I doing wrong?

### DISCUSSION THREAD

**Discussion by sicntired** [community/community#139338] [New to GitHub]
I have forgotten the reason I first went to GitHub?I just kept trying to open a discussion and nothiong works?The finder bar doesn’t print and there’s nowhere to begin any discussions?I tried clicking here and there and no joy.I’m an oldster and I have zero coding skills but can usually navigate the web easily.I understand GitHub is a developer site. I have used it before and I know sometimes I manage and others,like now,don’t work>?

**Comment by lqduy** [comment]
Thank you, sir, wishing you good health and happiness

**Comment by ghostinhershell** [comment]
Hi @sicntired, we're happy you're here! You are more likely to get a useful response if you are explicit about what your issue entails, giving a few more details might help someone give you a nudge in the right direction.
